wiki/Beatmaps/Packs/en.md

```diff
@@ -0,0 +1,48 @@
+# Beatmaps Packs
+
+_Main page: [Beatmaps](/wiki/Beatmaps)_
+
+Beatmaps packs, as the title suggests, are a `.zip` files containing beatmaps.
+The name of the pack is based on what they contain (e.g. `Approved Beatmap Pack #7` would only contain [approved](#approved) beatmaps).
+It is worth noting that the most packs are:
+
+- Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved)
+- Always contain videos (if included)
+- The beatmaps are either in the form of `.zip` or `.osz` file
```

Comment:
You can remove the `.zip` notice. It is for _ancient_-level beatmap packs which should be extinct by now.